Bottom: 8f4dd14fef8c75965652fe27e1913242bdbc773c
Top:    ad548d5bf94fb7327a0fbfa80527aca1c38cdff8
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-10 22:03:44 -0400

Refresh of tyler

---

diff --git a/sound/soc/generic/simple-card.c b/sound/soc/generic/simple-card.c
index 8840c36..588da3d 100644
--- a/sound/soc/generic/simple-card.c
+++ b/sound/soc/generic/simple-card.c
@@ -158,18 +158,6 @@ asoc_simple_card_sub_parse_of(struct device_node *np,
 				     "system-clock-frequency",
 				     &dai->sysclk);
 	} 
-/*
-else {
-		clkspec.np = node;
-		clk = of_clk_get_from_provider(&clkspec);
-
-		if (!IS_ERR(clk)) {
-			dai->sysclk = clk_get_rate(clk);
-			clk_put(clk);
-		}
-	}
-*/
-printk("JDS sysclk %d\n", dai->sysclk);
 	return 0;
 }
